Updated README - More info on the getting started section (#623)
python/python-docs-fr: README.rst

  if you don't already have one).

Getting Started:

You'll need to fork the `python-docs-fr
<https://github.com/python/python-docs-fr>`_ source repository by clicking its ``Fork``
button. This creates a copy of the whole project on your github
account: a place where you have the rights to do modifications.

Step by step:

.. code-block:: bash

    # Clone your github fork with `git` using ssh or https:
    git clone git@github.com:YOUR_GITHUB_USERNAME/python-docs-fr.git
    git clone https://github.com:YOUR_GITHUB_USERNAME/python-docs-fr.git

    # Go into the cloned directory:
    cd python-docs-fr/

    # Add the upstream (the public repository) using HTTPS (won't ask for password):
    # This is so git knows what/where the upstream is.
    git remote add upstream https://github.com/python/python-docs-fr.git

Next, you need to find a file to work on.
You can use `potodo <https://github.com/seluj78/potodo>`, a tool made to find ``po`` to do.
Install it using pip (``pip install potodo``) in a ``python3.7`` environement.

> Then run the command `potodo` in your cloned fork.

Reviewer: ```suggestion
Then run the command ``potodo`` in your cloned fork.
```

From the list returned by the command, you can choose any file that is not reserved.

**We recommend not starting with a file from ``c-api`` as it is very technical.**

Once you've choosen a file to work one, please open an `issue on github <https://github.com/python/python-docs-fr>` in the format of `I'm working on FOLDER/FILE.po`. This is done to update ``potodo`` as it checks the github API for reserved ``.po`` files in issues and pull requests.

Now you're ready to start a work session. Each time you'll start a new file, start here:

.. code-block:: bash

    # To work, we'll need a branch, based on an up-to-date (freshly fetched)

>     # upstream/3.7 branch. We will name our branch `library-sys` but you shall name yours

Reviewer: ```suggestion
    # upstream/3.7 branch. We will name our branch "library-sys" but you shall name yours
```

    # whatever you want. Usually you'll name a branch based on the file you're working on.

>     # For example, If you're working on `library/venv.po` you can name your branch `library-venv`

Reviewer: ```suggestion
    # For example, If you're working on "library/venv.po" you can name your branch "library-venv"
```

(on est dans un bloc de code donc ca ne sera pas rendu, et les gens ne lisent peut être pas naturellement le restructuredtext).

    
    # Update your local version to the latest
    git fetch upstream

>     # Create a new branch named `library-sys` based on `upstream/3.7`

Reviewer: ```suggestion
    # Create a new branch named "library-sys" based on "upstream/3.7"
```

    git checkout -b library-sys upstream/3.7

    # You can now work on the file, typically using poedit,

>     # Of course, replace `library/sys.po` by the file you've chose earlier

Reviewer: ```suggestion
    # Of course, replace "library/sys.po" by the file you've chose earlier
```

    poedit library/sys.po

>     # When you are done translating, you can run `pospell <https://github.com/JulienPalard/pospell>`.

Reviewer: ```suggestion
    # When you are done translating, you can run pospell (pip install pospell).
```

    # This tool was made to check if you don't have any french mistakes.

>     # You can run the following command: `pospell -p dict -l fr_FR **/*.po to check all files

Reviewer: ```suggestion
    # You can run the following command: pospell -p dict -l fr_FR **/*.po to check all files
```

>     # or replace `**/*.po` by your specific file (recommended).

Reviewer: ```suggestion
    # or replace **/*.po by your specific file (recommended).
```

>     # You can then run `powrap <https://github.com/JulienPalard/powrap>` which will rewrap

Reviewer: ```suggestion
    # You can then run powrap (pip install powrap) which will rewrap
```
